src/components/Body.jsx

```diff
@@ -0,0 +1,47 @@
+import Article from "./Article";
+import React from "react";
+import { useState, useEffect } from "react";
+import { LoadingSpinner } from "./LoadingSpinner";
+import axios from "axios";
+let url = "https://northcoders-news-be.onrender.com/api/articles";
+function Body() {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    axios
+      .get(url)
+      .then((response) => {
+        console.log(response.data);
+        setArticles(response.data);
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        console.log(error);
+        setIsLoading(false);
+      });
+  }, []);
+
+  if (isLoading) {
+    return <LoadingSpinner />;
+  }
+  return (
+    <>
```

Comment:
react fragments are only needed if there isn't a parent tag already which there is here with your div.